app/renderer/components/navigation/navigator.js

```diff
@@ -66,7 +65,7 @@ class Navigator extends ImmutableComponent {
         })
       }
     } else {
-      navAction.call(this.activeFrame)
+      navAction.call(this, this.props.activeTab.get('tabId'))
```

Comment:
we don't have this in yet. I didn't yet implement redux here, so we have what we have for now, but when I will add redux to it, we will have primitives.